timbot.py

```diff
@@ -107,5 +98,13 @@ def run_timbot():
 					else:
 						send_message('keep pounding')
 
+				# openstack meme                     #temp fix 
```

Comment:
Nested comments?

Reply:
the comment is nested because it only applies to the second half of the elif this will be removed with #34